python/cudf/cudf/io/orc.py

```diff
@@ -387,7 +387,16 @@ def read_orc_stripe(orc_file, stripe, columns):
 
 
 @ioutils.doc_to_orc()
-def to_orc(df, fname, compression=None, enable_statistics=True, **kwargs):
+def to_orc(
+    df,
+    fname,
+    compression=None,
+    enable_statistics=True,
+    stripe_size_bytes=67108864,
+    stripe_size_rows=1000000,
+    row_index_stride=10000,
```

Comment:
Can we update the public docstring for newly added params here: https://github.com/rapidsai/cudf/blob/27184431443fad3207590a10d9e32144d7c4c827/python/cudf/cudf/utils/ioutils.py#L382